src/main/scala/com/whisk/k8s/resolver/KubernetesClient.scala

```diff
@@ -63,23 +68,23 @@ class KubernetesClient(client: Service[Request, Response]) extends Closable {
     val service = endpoint.serviceName
     val url = s"/api/v1/namespaces/$namespace/endpoints/$service"
 
-    client(http.Request(http.Method.Get, url)).flatMap(response => {
-      val errorOrAddresses = K8sApiModel
-        .parseEndpoints(response.content)
-        .map(buildAddresses)
-      errorOrAddresses match {
-        case Left(error) => Future.exception(error)
-        case Right((version, addresses)) =>
-          Future.value((version, addresses))
-      }
-    })
+    client(http.Request(http.Method.Get, url)).flatMap { response =>
+      getContent(response)
+        .map(K8sApiModel.parseEndpoints)
+        .map(_.map(buildAddresses))
+        .flatMap {
+          case Left(error) => Future.exception(error)
+          case Right((version, addresses)) =>
+            Future.value((version, addresses))
+        }
+    }
   }
 
   /**
     * Watch changes for pods ip addresses for a endpoint of service defined in Kubernetes
     */
   def watchAddresses(endpoint: Endpoint, resourceVersion: String)(
-      callback: Seq[Address] => Unit): Future[_] = {
+    callback: Seq[Address] => Unit): Future[_] = {
```

Comment:
We use scalafmt for formatting, I don't think it's formatted correctly. Can you make sure, please.
You can select scalafmt as a default formatter, see [here](https://github.com/whisklabs/atlas#format-code) how.